target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c

```diff
@@ -316,6 +316,7 @@ static int __init rtl83xx_mdio_probe(struct rtl838x_switch_priv *priv)
 	for_each_node_by_name(dn, "port") {
 		phy_interface_t interface;
 		u32 led_set;
+		u32 led_num;
```

Comment:
This probably should be its own commit, though with the lack of a proper led driver atm I see what you are doing. Would this be working in any way with https://gitlab.com/olliver/openwrt/openwrt/-/commit/7df2e63f7bf2a1b7de18007e9cb209ea92e9c415 ? I know that that patch is a hack too; but both these are just while we wait for sander's led driver stuff ...